javascript/powercommand_using_SDKv3/task_client_filter.js

```diff
@@ -0,0 +1,63 @@
+const filterDefs = {
+  all: {
+    key: 'all',
+    matchesTask: () => true,
+  },
+  name: {
+    key: 'name',
+    value: null,
+    matchesTask: function(task) {
+      return task.name && task.name.includes(this.value);
+    },
+  },
+  notes: {
+    key: 'notes',
+    value: null,
+    matchesTask: function(task) {
+      return task.notes && task.notes.includes(this.value);
+    },
+  },
+  assignee: {
+    key: 'assignee',
+    value: null,
+    matchesTask: function(task) {
+      return task.assignee && task.assignee.name.includes(this.value);
```

Comment:
You might want to add another check in case the task has an `assignee` but no `assignee.name`. This was probably broken before as well since [`client.projects.tasks(projectId);`](https://github.com/Asana/devrel-examples/blob/master/javascript/powercommands/task_provider.js#L3) does not return the full task object with assignee
`return task.assignee && task.assignee.name.includes(this.value);` ->
`return task.assignee && task.assignee.name && task.assignee.name.includes(this.value);`

Reply:
Added this